R/trim-osmdata.R

```diff
@@ -155,6 +155,10 @@ trim_to_poly_pts <- function (dat, bb_poly, exclude = TRUE) {
 
     if (is (dat$osm_points, "sf")) {
 
+        if (!requireNamespace ("sp", quietly = TRUE)) {
+            stop ('trim_osmdata requires sp package:\n\tinstall.packages("sp")')
```

Comment:
As in `elevation.R`, better to just have a simple call to `requireNamespace("sp")`.

Reply:
Done